server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java

```diff
@@ -135,10 +142,15 @@ public boolean userEditable() {
 
 	@Override
 	public void dataTick() {
+		timeAtLastUpdate = System.currentTimeMillis();
 	}
 
 	@Override
 	public void tick() {
+		if (useTimeout) {
+			if (System.currentTimeMillis() - timeAtLastUpdate > TIMEOUT_MS)
+				setStatus(TrackerStatus.DISCONNECTED);
```

Comment:
add the reverse of it? it'll always be disconnected

Reply:
The tracker will never stop being disconnected and will be set to DISCONNECTED again on each tick.

Reply:
timeAtLastUpdate gets updated to System.currentTimeMillis, so if it's receiving data it'll be for example:
19348 - 19348 > 3000
0 > 3000
false

If not, it'll be, for example:
29348 - 19348 > 3000
10000 > 3000
true

Reply:
Okay but where does it get out of the disconnected state?

Reply:
Trackers received from VMC and feeder app are changed to OK on each data update, but I'll double check HMD later

Reply:
Yes HMD sets it to OK... I remembered which part of my code did that and saw you reviewed it asking for a comment :p

Reply:
Resolving this, will rewrite trackers to be better as my next big PR anyway...